runelite-client/src/main/java/net/runelite/client/config/RuneLiteConfig.java

```diff
@@ -244,11 +244,24 @@ default FlashNotification flashNotification()
 		return FlashNotification.DISABLED;
 	}
 
+	@ConfigItem(
+		keyName = "flashNotificationDuration",
+		name = "Flash notification duration",
+		description = "Sets the duration of the flash notification",
+		position = 25,
+		section = notificationSettings
+	)
+	@Units(Units.MILLISECONDS)
+	default int flashNotificationDuration()
+	{
+		return 2000;
+	}
+
 	@ConfigItem(
 		keyName = "notificationFocused",
 		name = "Send notifications when focused",
 		description = "Toggles all notifications for when the client is focused",
-		position = 25,
+		position = 27,
```

Comment:
You're reordering the existing focused/flash color options here. Instead this should be 26 and the next one 27.